pipelines/nextflow/workflows/nextflow.config

```diff
@@ -39,9 +39,8 @@ trace {
     // filter = 'name =~ /foo.*/ && status == "FAILED"'
 }
 
-notification {
+singularity {
```

Comment:
We need to remember to make `singularity` a dependency of the Nextflow environment in this case.

Reply:
We need to have either docker or singularity here, otherwise the scripts called need to be in the local environment